resources/qml/EncryptionIndicator.qml

```diff
@@ -0,0 +1,24 @@
+import QtQuick 2.5
+import QtQuick.Controls 2.1
+import com.github.nheko 1.0
```

Comment:
General comment for all of these... should we use 'im.nheko' instead of 'com.github.nheko'?  Since I have the nheko.im registered.  Not really a big deal... just a thought.

Reply:
Yeah, that sounds reasonable, I'll do that